charts/ark-solr/templates/statefulset.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "solr.fullname" . }}-statefulset
+  labels:
+    {{- include "solr-helm-charts.labels" . | nindent 6 }}
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  replicas: 1
+  selector: 
+    matchLabels:  
+      {{- include "solr-helm-charts.labels" . | nindent 6 }}
+  serviceName: {{ include "solr.fullname" . }}-headless
+  template:
+    metadata:
+      name: ark-solr
```

Comment:
@naveen-armedia  Do you need to specify the `name` here? I think you can skip this and let Kubernetes name the pods according to its StatefulSet scheme.

Reply:
@fabricetriboix I would prefer having name here, so that it would be more readable and clear when checking about the app status later when we are debugging.

Reply:
@naveen-armedia Would Kubernetes postfix the name with `-0`, `-1`, etc. for the various pods of the StatefulSet in that case?

Reply:
@fabricetriboix Yes, after postfix name with `-0`, `-1` pod name looks like `ark-solr-statefulset-0`. I think without name it would be like  `statefulset-0`.

Reply:
All right, thanks @naveen-armedia